pages/stable-version.html

```diff
@@ -173,23 +173,39 @@ <h5>Quickstart Installation Instructions</h5>
                 <p>Please note Uyuni requires FQDNs for the Server, Proxy and clients, so usage of a properly-configured DNS server is mandatory for client updates to work.</p>
                 <p>If you want to install Uyuni in lab mode, for testing, you may be interested in how to easily make use of libvirt's included dnsmasq DNS and DHCP server: <a href="https://github.com/uyuni-project/uyuni/wiki/Libvirt-DNS-and-DHCP-without-Avahi">Libvirt DNS and DCHP without Avahi</a>.</p>
 
-                <p>1. Update openSUSE Leap 15.5:</p>
-                <code>$ sudo zypper update</code>
+                <p>1. Update openSUSE Leap Micro 5.5:</p>
+                <code>$ transactional update</code>
```

Comment:
Same as before

Reply:
Also here, 177, 184, 190, 193 and 198 should use `#` if I am not wrong.

Reply:
Seems you missed the change here?

Reply:
it should be all resolved now